src/Utopia/Messaging/Adapters/SMS/Telesign.php

```diff
@@ -50,8 +50,18 @@ protected function process(SMS $message): string
             ],
             body: \http_build_query([
                 'template' => $message->getContent(),
-                'recipients' => \implode(',', $to),
+                'recipients' => $to,
             ]),
         );
     }
+
+    private function formatNumbers(array $numbers): string
+    {
+        $formatted = \array_map(
+            fn ($number) => $number.':'.\uniqid(),
```

Comment:
Why do we add a unique ID here? Don't we still need to trim the `+` as well?

Reply:
The documentation recommended adding the id, but it is optional. Do you want to remove it? Also the number format was fine with the +. I can try to remove to see if it makes a difference